### ISSUE TITLE
OSCE pitch.py hangover function useless

### ISSUE THREAD

**Issue by ArameMao** [xiph/opus#305]
I found that the value of "last_lag" in function [hangover()](https://github.com/xiph/opus/blob/6d7ae213ce07cd3090a3d77ad0e79035bcf5d191/dnn/torch/osce/utils/pitch.py#L32-L47) never changes, this function seems useless:
```
def hangover(lags, num_frames=10):
    lags = lags.copy()
    count = 0
    last_lag = 0

    for i in range(len(lags)):
        lag = lags[i]

        if lag == 0:
            if count < num_frames:
                lags[i] = last_lag
                count += 1
        else:
            count = 0

    return lags
```

**Comment by janpbuethe** [comment]
Thanks for reporting this. A fix is currently under evaluation in opus-ng-lace-integration-model-tuning and will be merged back soon.

**Comment by janpbuethe** [comment]
Fixed with commit 367a487e7.
